### DISCUSSION TITLE
Dcucumber.filter.tags Not running any test cases

### DISCUSSION THREAD

**Discussion by psrini7** [serenity-bdd/serenity-core#3100] [Q&A]
Question: I understand many had this issue but I tried hard to fix but still I’m unable to run a test using
mvn clean verify -Dcucumber.filter.tags="@smoke" 0 test runs.

-Dtags="@smoke" - this works though but only with one tag.

Kindly let me know if anyone had fixed this issue in past.
I’m using serenity 3.5.1 version.

**Comment by ndagnhat** [comment]
Try it: mvn clean verify -D"cucumber.filter.tags='@smoke'"

It work for me
